oximeter/collector/src/agent.rs

```diff
@@ -183,13 +233,14 @@ impl OximeterAgent {
         db_config: Option<DbConfig>,
         log: &Logger,
     ) -> Result<Self, Error> {
-        let (result_sender, result_receiver) = mpsc::channel(8);
         let log = log.new(o!(
             "component" => "oximeter-standalone",
             "collector_id" => id.to_string(),
             "collector_ip" => address.ip().to_string(),
         ));
 
+        let collection_task_wrapper = CollectionTaskWrapper::new();
```

Comment:
If we are only using a single node here, we can probably use the `mpsc::channel directly`.

Reply:
We can't really because `OximeterAgent.result_sender` is of type `CollectionTaskSenderWrapper`. Later, it's `send()` method is used by  `CollectionTask`'s `run()` method. I would have to modify this `run()` method to make it somehow aware of single node vs cluster. 

Frankly, I would rather not do that since this code is all going to go away when we no longer run a single node installation, and this standalone oximeter is only used for testing.

Reply:
Makes sense.